ghcide/src/Development/IDE/Plugin/HLS.hs

```diff
@@ -177,9 +191,9 @@ executeCommandHandlers recorder ecs = requestHandler SMethod_WorkspaceExecuteCom
                 -- If we have a command, continue to execute it
                 Just (Command _ innerCmdId innerArgs)
                     -> execCmd ide (ExecuteCommandParams Nothing innerCmdId innerArgs)
-                Nothing -> return $ Right $ InL A.Null
+                Nothing -> return $ Right $ InR Null
```

Comment:
This is fine but I it's literally the same behaviour. The upstream spec is weird, I made an issue: https://github.com/microsoft/language-server-protocol/issues/1766